lib/consoletest.pm

```diff
@@ -51,6 +58,31 @@ sub post_fail_hook {
     $self->export_logs_desktop;
 }
 
+=head2 get_avc_selinux_alerts
+
+List AVCs that have been recorded during a runtime of a test module that executes this function
+
+=cut
+
+sub get_avc_selinux_alerts {
```

Comment:
The name `get_avc_selinux_alerts` suggests that something will be returned, which is not the case when messages are retrieved from the AVC.

Given how I **think** this subroutine was intended, I would rename it to `record_selinux_alerts` and replace the `return undef` with simple `return`, as the result is anyways not used anywhere.